### ISSUE TITLE
Problems with buttons and icons

### ISSUE THREAD

**Issue by TorakikiSan** [olton/Metro-UI-CSS#1648]
Hi Olton!

After updating to Metro 4.4.1, I have an icon button display problem. I attach image

![SharedScreenshot](https://user-images.githubusercontent.com/37299746/96520093-9cb8ee80-126e-11eb-84c8-0e55e55fcea1.jpg)

See also

https://codepen.io/TorakikiSan/pen/EeyWmp

The buttons with only the icon, have these shifted to the left and the buttons with icon and text are attached without the space that is in the HTML code and that was previously displayed.

The problem is on both Firefox and Chrome

Thank you!

**Comment by olton** [comment]
I'll fix it in 4.4.2, for now use this fix:
```
.button.square, .button.cycle {
  justify-content: center;
}
```

**Comment by TorakikiSan** [comment]
Thank You!
